docs/stellar-core_example.cfg

```diff
@@ -609,6 +620,51 @@ LOADGEN_OP_COUNT_DISTRIBUTION_FOR_TESTING=[]
 # the core's internal errors via catching them early.
 HALT_ON_INTERNAL_TRANSACTION_ERROR=false
 
+# ENABLE_SOROBAN_DIAGNOSTIC_EVENTS defaults to false.
+# If set to true, additional diagnostic Soroban events that are not part
+# of the protocol will be generated while applying Soroban transactions.
+# These events will be put into a list in the non-hashed portion of the
+# meta, and this list will contain all events so ordering can be
+# maintained between all events. The default value is false, and this
+# should not be enabled on validators.
```

Comment:
nit: 'should not' -> 'can not' - the validator node will crash if this is enabled.

Reply:
Fixed